apps/chronicles/screens/components/SwitchScreen.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { ScrollView, View } from "react-native";
+import { EDSStyleSheet, Spacer, Typography, useStyles } from "@equinor/mad-components";
+import { Switch } from "../../../../packages/components/src/components/Switch/";
```

Comment:
This import should be pointing to the package and not relative reference in the monorepo.

Reply:
It is probably why we dont get dark mode support btw!